sys/net/application_layer/dns/msg.c

```diff
@@ -7,31 +7,24 @@
  */
```

Comment:
Just to avoid confusion in the review: I did not rename the file and then moved the `sock_dns` parts to `sys/net/application_layer/dns/dns.c`, rather I renamed `sys/net/application_layer/dns/dns.c` to `sys/net/application_layer/sock_dns/dns.c` and then moved the message composition and parsing parts from `sys/net/application_layer/sock_dns/dns.c` to this `sys/net/application_layer/dns/msg.c`. However, since that was the majority of `sys/net/application_layer/sock_dns/dns.c`'s content, Git interpreted it as a rename.